Readme.md

```diff
@@ -85,7 +85,7 @@ https://download.respondus.com/lockdown/download7.php?id=XXXXXXXXX
 If the _Lockdown Browser_ fails to launch, **do not double-click** the file on the VM's desktop. Instead, open a PowerShell prompt inside the VM and run:
 
 ```powershell
-cd C:\Users\WDAGUtilityAccount\Desktop\runtime_directory\
+Set-Location C:\Users\WDAGUtilityAccount\Desktop\runtime_directory\
```

Comment:
Changing the aliases to the long-form versions makes sense for the scripts, but since this is meant to be typed in interactively, I'd leave this as the shorter `cd`.
```suggestion
cd C:\Users\WDAGUtilityAccount\Desktop\runtime_directory\
```